docs/DeveloperGuide.md

```diff
@@ -423,7 +436,41 @@ Actor: User
 
      Use case resumes at step 2
 
-**Use Case: Upload Image**
+### **Use Case: Find clients by keywords**
+
+Actor: User
+
+**MSS**
+1. User requests to find clients.
+2. REP filters the full list of clients for those that satisfy the condition.
+3. REP shows the filtered list of clients. 
+
+**Extensions:**
+* 1a. User request does not specify an attribute. 
+  * 1a1. REP displays an error message.
+  
+    Use case ends.
+* 1b. User request does not specify any keywords.
+  * 2b1. REP displays an error message.
```

Comment:
Maybe this two can be combined to be `user request has wrong details` since use cases are not meant to describe the internal details of a system and only the externally visible behaviour

Reply:
Changes made.